Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -244,11 +257,24 @@ extension BrowserViewController {
     present(popup, animated: false)
   }
   
-  private func presentVPNChurnPromoCallout(for type: VPNChurnPromoType) {
+  private func presentVPNUpdateBillingCallout(skipSafeGuards: Bool = false) {
+    if !skipSafeGuards {
+      // TODO: Condition
+      return
+    }
+    
+    presentVPNChurnPromoCallout(for: .updateBillingExpired) {
+      // TODO: Action
```

Comment:
Let's open iOS 'manager subscription' url here.
Unless you find a better url to directly update payment info

Reply:
I will take care of action part and condition of presentation in upcoming PR.